src/server.c

```diff
@@ -133,10 +148,7 @@ static bool handle_server_startup(PgSocket *server, PktHdr *pkt)
 		break;
 
 	case 'E':		/* ErrorResponse */
-		if (!server->pool->welcome_msg_ready)
-			kill_pool_logins_server_error(server->pool, pkt);
-		else
-			log_server_error("S: login failed", pkt);
+		kill_pool_logins_server_error(server->pool, pkt);
```

Comment:
Why to remove the below log?
`log_server_error("S: login failed", pkt);`

Reply:
We now always call `kill_pool_logins_server_error` and that already logs the error using:
```c
log_warning("server login failed: %s %s", level, msg);
```

(it's a warning not an error this time, but that seems like a not very important change)